### DISCUSSION TITLE
qDate, qTime close PopUp proxy on select date / time

### DISCUSSION THREAD

**Discussion by bitmandk** [quasarframework/quasar#15393] [General - Components / Directives / etc]
Using qData and qTime with Popup proxy, I want to close the Popup proxy upon selection, found a few older examples using @input but can't find @input in the documentation anymore for qDate - any alternatives to get this to work?

`<q-popup-proxy ref="dateProxy" transition-show="scale" transition-hide="scale">`
`<q-date @input="$refs.dateProxy.hide()" v-model="date" mask="YYYY-MM-DD HH:mm">`

[CodePen](https://codepen.io/bobittjek/pen/abjMeNO?editors=1010)

**Comment by metalsadman** [comment]
In essence it's still the same, use template ref. See vue 3 docs how.

**Reply by bitmandk** [reply]
How, as @input is not in the Quasar documentation for Data and Time component, as far as I can see.

**Reply by metalsadman** [reply]
@input = @update:model-value, please see vue 3 migration guide.

**Comment by MilosPaunovic** [comment]
https://codepen.io/milospaunovic/pen/WNKWdgP?editors=1010

But, as Aldrin said, it's crucial to go through the `Vue@3` migration guide for these kind of issues.
https://quasar.dev/start/upgrade-guide
